docs/recipes/Mocking-API-Calls.md

```diff
@@ -2,35 +2,87 @@
 
 # Mocking your API calls for Local Development
 
-<!-- TODO: expand on this section -->
 
-The `set-middleware` command links your module's custom dev middleware file to One App:
+1. Setup the API using [making an API guide](../../recipes/Making-An-API-Call.md). The API call would be similar to this
 
-```bash
-$ npm run set-middleware ../[module-name]/dev.middleware.js
-```
+    ```javascript
+    const loadModuleData = async ({ store, fetchClient }) => {
+      const moduleState = store.getState().getIn(['modules', 'my-first-module']);
+      const fastRes = await fetchClient(store.getState().getIn(['config', 'oneAppApiUrl']));
```

Comment:
maybe change the `oneAppApiUrl` example name to `someApiUrl`. The first one sounds like it is a special name when in reality it could be anything